src/core/Carousel/CarouselDots.tsx

```diff
@@ -0,0 +1,138 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import cx from 'classnames';
+import { CarouselContext } from './CarouselContext';
+import { useTheme } from '../utils';
+import { CarouselDot } from './CarouselDot';
+import '@itwin/itwinui-css/css/carousel.css';
+
+type CarouselDotsProps = {
+  /** Number of total dots/slides in the carousel. Will be inferred from Carousel context. */
+  length?: number;
+  /** Maximum number of dots to show in the viewport. @default 5 */
+  maxCount?: number;
+  /** Index of currently acitve dot. Will be inferred from Carousel context. */
```

Comment:
acitve

Can you install spell checker in vscode please? :D

Reply:
![image](https://user-images.githubusercontent.com/9084735/159265922-c0b65a25-2419-45a5-aecd-96415ee9248b.png)

Reply:
so it didnt mark or whats the deal?

Reply:
Reinstalled and it's working now